app/src/main/res/layout/profile_chooser_profile_view.xml

```diff
@@ -28,19 +31,20 @@
 
     <TextView
       android:id="@+id/profile_name_text"
-      android:layout_width="wrap_content"
+      android:layout_width="112dp"
       android:layout_height="wrap_content"
       android:fontFamily="sans-serif-medium"
-      android:text="@{viewModel.profile.name}"
```

Comment:
Why not keep the data-binding? If you ellipsize here, you can keep the data-binding.

Reply:
Because of that ellipsis not working with max limit. Will remove this as you want me to remove that max limit.

Reply:
Done.